src/envoy/http/authn/testdata/envoy_origin_jwt.conf

```diff
@@ -30,24 +30,36 @@
               }
             ],
             "filters": [
+              {
+                "type": "decoder",
+                "name": "jwt-auth",
```

Comment:
Do you need this? I don't think you use this file at all.

Reply:
Yes, I use this config file to do non-mocked test.

Reply:
When running tests with this config file, in one terminal, I run "bazel-bin/src/envoy/envoy -c src/envoy/http/authn/testdata/envoy_origin_jwt.conf". In another terminal, I run: curl --header "Authorization: Bearer $token" http://localhost:9090/echo -d "hello world"

Reply:
I don't see it use in this PR. Also, it sounds like you have to link with jwt_auth lib in order to do that. It doesn't sound right.

Reply:
This file has been removed.